warning: The practice of declaring the Bind placeholders (_1, _2, ...) in the global namespace is deprecated

```
/Users/rex/zcash/depends/x86_64-apple-darwin19.6.0/share/../include/boost/bind.hpp:36:1: warning: The practice of declaring the Bind placeholders (_1, _2, ...) in the global namespace is deprecated. Please use <boost/bind/bind.hpp> + using namespace boost::placeholders, or define BOOST_BIND_GLOBAL_PLACEHOLDERS to retain the current behavior. [-W#pragma-messages]
BOOST_PRAGMA_MESSAGE(
^
/Users/rex/zcash/depends/x86_64-apple-darwin19.6.0/share/../include/boost/config/pragma_message.hpp:24:34: note: expanded from macro 'BOOST_PRAGMA_MESSAGE'
# define BOOST_PRAGMA_MESSAGE(x) _Pragma(BOOST_STRINGIZE(message(x)))
                                 ^
<scratch space>:96:2: note: expanded from here
 message("The practice of declaring the Bind placeholders (_1, _2, ...) " "in the global namespace is deprecated. Please use " "<boost/bind/bind.hpp> + using namespace boost::placeholders, " "or define BOOST_BIND_GLOBAL_PLACEHOLDERS to retain the current behavior.")
 ^
1 warning generated.
```

No, the patch causes a build failure.

```
/Users/rex/zcash/depends/x86_64-apple-darwin19.6.0/share/../include/boost/bind/bind.hpp:259:9: error: no matching function for call to object of type 'std::__1::function<void (const CRPCCommand &)>'
        unwrapper<F>::unwrap(f, 0)(a[base_type::a1_]);
        ^~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/rex/zcash/depends/x86_64-apple-darwin19.6.0/share/../include/boost/bind/bind.hpp:1306:16: note: in instantiation of function template specialization 'boost::_bi::list1<boost::_bi::value<std::__1::placeholders::__ph<1> > >::operator()<std::__1::function<void (const CRPCCommand &)>, boost::_bi::rrlist1<const CRPCCommand &> >' requested here
        return l_( type<result_type>(), f_, a, 0 );
               ^
/Users/rex/zcash/depends/x86_64-apple-darwin19.6.0/share/../include/boost/function/function_template.hpp:158:33: note: in instantiation of function template specialization 'boost::_bi::bind_t<boost::_bi::unspecified, std::__1::function<void (const CRPCCommand &)>, boost::_bi::list1<boost::_bi::value<std::__1::placeholders::__ph<1> > > >::operator()<const CRPCCommand &>' requested here
          BOOST_FUNCTION_RETURN((*f)(BOOST_FUNCTION_ARGS));
                                ^
/Users/rex/zcash/depends/x86_64-apple-darwin19.6.0/share/../include/boost/function/function_template.hpp:940:53: note: in instantiation of member function 'boost::detail::function::void_function_obj_invoker1<boost::_bi::bind_t<boost::_bi::unspecified, std::__1::function<void (const CRPCCommand &)>, boost::_bi::list1<boost::_bi::value<std::__1::placeholders::__ph<1> > > >, void, const CRPCCommand &>::invoke' requested here
        { { &manager_type::manage }, &invoker_type::invoke };
                                                    ^
/Users/rex/zcash/depends/x86_64-apple-darwin19.6.0/share/../include/boost/function/function_template.hpp:720:13: note: in instantiation of function template specialization 'boost::function1<void, const CRPCCommand &>::assign_to<boost::_bi::bind_t<boost::_bi::unspecified, std::__1::function<void (const CRPCCommand &)>, boost::_bi::list1<boost::_bi::value<std::__1::placeholders::__ph<1> > > > >' requested here
      this->assign_to(f);
            ^
/Users/rex/zcash/depends/x86_64-apple-darwin19.6.0/share/../include/boost/function/function_template.hpp:1086:5: note: in instantiation of function template specialization 'boost::function1<void, const CRPCCommand &>::function1<boost::_bi::bind_t<boost::_bi::unspecified, std::__1::function<void (const CRPCCommand &)>, boost::_bi::list1<boost::_bi::value<std::__1::placeholders::__ph<1> > > > >' requested here
    base_type(f)
    ^
/Users/rex/zcash/depends/x86_64-apple-darwin19.6.0/share/../include/boost/function/function_template.hpp:1139:5: note: in instantiation of function template specialization 'boost::function<void (const CRPCCommand &)>::function<boost::_bi::bind_t<boost::_bi::unspecified, std::__1::function<void (const CRPCCommand &)>, boost::_bi::list1<boost::_bi::value<std::__1::placeholders::__ph<1> > > > >' requested here
    self_type(f).swap(*this);
    ^
/Users/rex/zcash/depends/x86_64-apple-darwin19.6.0/share/../include/boost/signals2/detail/slot_template.hpp:160:24: note: in instantiation of function template specialization 'boost::function<void (const CRPCCommand &)>::operator=<boost::_bi::bind_t<boost::_bi::unspecified, std::__1::function<void (const CRPCCommand &)>, boost::_bi::list1<boost::_bi::value<std::__1::placeholders::__ph<1> > > > >' requested here
        _slot_function = detail::get_invocable_slot(f, detail::tag_type(f));
                       ^
/Users/rex/zcash/depends/x86_64-apple-darwin19.6.0/share/../include/boost/signals2/detail/slot_template.hpp:85:9: note: in instantiation of function template specialization 'boost::signals2::slot<void (const CRPCCommand &), boost::function<void (const CRPCCommand &)> >::init_slot_function<boost::_bi::bind_t<boost::_bi::unspecified, std::__1::function<void (const CRPCCommand &)>, boost::_bi::list1<boost::_bi::value<std::__1::placeholders::__ph<1> > > > >' requested here
        init_slot_function(f);
        ^
rpc/server.cpp:66:37: note: in instantiation of function template specialization 'boost::signals2::slot<void (const CRPCCommand &), boost::function<void (const CRPCCommand &)> >::slot<boost::_bi::bind_t<boost::_bi::unspecified, std::__1::function<void (const CRPCCommand &)>, boost::_bi::list1<boost::_bi::value<std::__1::placeholders::__ph<1> > > > >' requested here
    g_rpcSignals.PreCommand.connect(boost::bind(slot, _1));
                                    ^
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/functional:2410:9: note: candidate function not viable: no known conversion from 'std::__1::placeholders::__ph<1>' to 'const CRPCCommand' for 1st argument
    _Rp operator()(_ArgTypes...) const;
        ^
3 errors generated.
```

--- a/src/init.cpp
+++ b/src/init.cpp
@@ -56,7 +56,7 @@
 #include <boost/algorithm/string/predicate.hpp>
 #include <boost/algorithm/string/replace.hpp>
 #include <boost/algorithm/string/split.hpp>
-#include <boost/bind.hpp>
+#include <boost/bind/bind.hpp>
 #include <boost/interprocess/sync/file_lock.hpp>
 #include <boost/thread.hpp>
 
@@ -67,6 +67,7 @@
 #include "librustzcash.h"
 
 using namespace std;
+using namespace boost::placeholders;
 
 extern void ThreadSendAlert();
 

--- a/src/rpc/server.cpp
+++ b/src/rpc/server.cpp
@@ -18,7 +18,7 @@
 
 #include <univalue.h>
 
-#include <boost/bind.hpp>
+#include <boost/bind/bind.hpp>
 #include <boost/filesystem.hpp>
 #include <boost/foreach.hpp>
 #include <boost/iostreams/concepts.hpp>
@@ -32,6 +32,7 @@
 
 using namespace RPCServer;
 using namespace std;
+using namespace boost::placeholders;
 
 static bool fRPCRunning = false;
 static bool fRPCInWarmup = true;

--- a/src/torcontrol.cpp
+++ b/src/torcontrol.cpp
@@ -14,7 +14,7 @@
 #include <set>
 #include <stdlib.h>
 
-#include <boost/bind.hpp>
+#include <boost/bind/bind.hpp>
 #include <boost/signals2/signal.hpp>
 #include <boost/foreach.hpp>
 #include <boost/algorithm/string/predicate.hpp>
@@ -27,6 +27,8 @@
 #include <event2/util.h>
 #include <event2/event.h>
 #include <event2/thread.h>
+
+using namespace boost::placeholders;
 
 /** Default control port */
 const std::string DEFAULT_TOR_CONTROL = "127.0.0.1:9051";

